pkg/xds/conv/convertxds.go

```diff
@@ -522,7 +522,7 @@ func convertIstioPercentage(percent *xdstype.FractionalPercent) uint32 {
 	case xdstype.FractionalPercent_MILLION:
 		return percent.Numerator / 10000
 	case xdstype.FractionalPercent_TEN_THOUSAND:
-		return percent.Numerator / 100
+		return percent.Numerator / 10
```

Comment:
why changed the `100` to `10` ?

Reply:
because the result is percent，if denominator is thousand, the percent will except 10 not 100

Reply:
TEN_THOUSAND

Reply:
sorry, i read it wrong